frontend/src/main/pages/PlayPage.js

```diff
@@ -61,64 +60,52 @@ export default function PlayPage() {
     );
   // Stryker enable all 
```

Comment:
Please change to `restore`; see: https://ucsb-cs156-s23.slack.com/archives/C050779M19P/p1685726928710669